libs/report/ingestion/src/lib/send-manifest.ts

```diff
@@ -0,0 +1,116 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client/core';
+import { readFileSync, statSync } from 'node:fs';
+import { TELEMETRY_INITIALIZE_MUTATION, TELEMETRY_REPORT_MUTATION } from './queries';
+import { type JSONValue } from './types';
+import { promptToProceedUploadFile } from './prompts';
+
+const client = new ApolloClient({
+  cache: new InMemoryCache(),
+  uri: 'https://api.nes.herodevs.com/graphql',
```

Comment:
let's make this configurable, either by an env var or `<some other option the CLI has?>`

Reply:
Also, could we include a user-agent header with something like `hdcli/<version>` in the headers?

Reply:
made configurable by env var and added user agent